config/casestudies/adeo.yml

```diff
@@ -263,3 +263,310 @@ asyncapi:
     * [AsyncAPI JavaScript Parser](https://github.com/asyncapi/parser-js) with [Avro Schema Parser](https://github.com/asyncapi/avro-schema-parser).
   fullExample: resources/casestudies/adeo/asyncapi.yaml
 additionalResources: Watch [this video presentation about AsyncAPI case study](https://www.youtube.com/watch?v=WwhRbvrf6Rs) from Ludovic Dussart, Ineat & Antoine Delequeuche, Adeo.
+
+examples: |
```

Comment:
we cannot solve it this way as this is content duplication, and we definitely should avoid it

we should read content from https://github.com/asyncapi/website/blob/master/public/resources/casestudies/adeo/asyncapi.yaml maybe inside script or directly in `[id].js`, but script would be better imho. 

`/resources/casestudies/adeo/asyncapi.yaml`:
- `/resources/casestudies/` is static
- `/adeo` is filename of
- `asyncapi.yaml` it will always be like that, although your code can support `*asyncapi.yml` and `*asyncapi.yaml` so there can be multiple files with the `asyncapi` in it

Reply:
okay I will look into it